share/man/man7/c.7

```diff
@@ -211,17 +212,120 @@ It incorporates the Technical Corrigendum 1 (ISO/IEC 9899:2011/COR1:2012)
+check resource availability with preprocessor directives
+.It
+Add the #warning directive for diagnostics
+.It
+Add the #elifdef and #elifndef directives
+.It
+Add the u8 prefix for character literals to represent UTF-8 encoding,
+compatible with C++17
+.It
+Add the char8_t type for UTF-8 encoded data and update the types of u8
+character constants and string literals to char8_t
+.It
+Add functions mbrtoc8() and c8rtomb() to convert between narrow multibyte
+characters and UTF-8 encoding
+.It
+Define all char16_t strings and literals as UTF-16 encoded and char32_t
+strings and literals as UTF-32 encoded unless specified otherwise
+.It
+Allow storage-class specifiers within compound literals
+.It
+Support the latest IEEE 754 standard, ISO/IEC 60559:2020, with binary and
+(optional) decimal floating-point arithmetic
+.It
+Add single-argument _Static_assert for compatibility with C++17
+.It
+Add _Decimal32, _Decimal64, _Decimal128 keywords for (optional) decimal
+floating-point arithmetic
+.It
+Add digit separator ' (the single quote character) for literals
+.It
+Enable specification of the underlying type of an enum
+.It
+Standardise the typeof() operator
+.It
+Add memset_explicit() in <string.h> to securely erase sensitive data
+regardless of optimisations
+.It
+Add memccpy() in <string.h> for efficient string concatenation
+.It
+Add memalignment() in <stdlib.h> to determine pointer alignment
+.It
+Add strdup() and strndup() in <string.h> to allocate string copies
+.It
+Introduce bit utility functions, macros, and types in the new header
+<stdbit.h>
+.It
+Add timegm() in <time.h> for converting time structures to calendar time
+values
+.It
+Add __has_include for header availability checking via preprocessor
+directives
+.It
+Add __has_c_attribute to check attribute availability via preprocessor
+directives
+.It
+Add _BitInt(N) and unsigned _BitInt(N) for bit-precise integers, and
+BITINT_MAXWIDTH for maximum bit width
+.It
+Elevate true and false to proper keywords (previously macros from
+<stdbool.h>)
+.It
+Add keywords alignas, alignof, bool, static_assert, thread_local; previously
+defined keywords remain available as alternative spellings
+.It
+Enable zero initialisation with {} (including initialisation of VLAs)
+.It
+Introduce C++11 style attributes using [[]], and support [[deprecated]],
+[[fallthrough]], [[maybe_unused]], [[nodiscard]], and [[noreturn]] for
+compatibility (all standard attributes can also be surrounded by double
+underscores, e.g. [[__noreturn__]])
+.It
+Deprecate _Noreturn, noreturn, header <stdnoreturn.h> features introduced
+in C11
+.It
+Remove trigraph support
+.It
+Remove K&R function definitions and declarations
+.It
+Remove non-two's-complement representations for signed integers
+.El
 .Pp
-Some useful features have been provided as extensions by some compilers, but
-they cannot be considered as standard features.
+The next version of the C Standard, informally named C2Y, is anticipated
```

Comment:
"C2x" used above. Pick one case?

Reply:
Sorry, I didn't quite catch.

Reply:
C2x vs C2X, pick upper or lower for all uses.

Reply:
Oh, I see! Silly me, LOL.